AC3.2Unit2FinalAssessment/AC3.2Unit2FinalAssessment/ColorTableViewController.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  ColorTableViewController.swift
+//  AC3.2Unit2FinalAssessment
+//
+//  Created by Eashir Arafat on 10/6/16.
+//  Copyright © 2016 Evan. All rights reserved.
+//
+
+import UIKit
+
+class ColorTableViewController: UITableViewController {
+    
+    var crayons: [Crayon] = []
+    
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // crayolaColors is a global variable defined in CrayonData.swift
+        // so it should be visible from any point in your code
+        
+        for c in crayolaColors {
+            if let crayon = Crayon(fromDict: c) {
+                crayons.append(crayon)
+            }
+        }
+
+        
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+
+        // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+        // self.navigationItem.rightBarButtonItem = self.editButtonItem()
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+  
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        // #warning Incomplete implementation, return the number of rows
+        return crayons.count
+    }
+
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "colorCell", for: indexPath)
+        
+        // Configure the cell...
```

Comment:
Easier/cleaner to assign the crayon object to a variable.
`
let crayon = crayons[indexPath.row]
then you can access the properties like crayon.name , crayon.blue etc...
`